classes/order/OrderInvoice.php

```diff
@@ -196,7 +196,6 @@ public function getProducts($products = false, $selected_products = false, $sele
             }
 
             $this->setProductImageInformations($row);
```

Comment:
```suggestion
            $product['current_stock'] = '--';
```
Shouldn't it be this for the equivalence? Unless the `current_stock` field is not useful anymore

Reply:
@jolelievre I will check it, I think I removed it because it wasnt used anywhere.

Reply:
@jolelievre It wasn't used anywhere, but I added the info in my last commit. :-)